CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.23.3
+
+* Upgrade to nginx 1.23.3.
```

Comment:
Could you include some of the other changes that may be material?

Reply:
All changes are related to upgrading.